README.md

```diff
@@ -67,6 +67,10 @@ of the OpenSSL toolkit are available. In particular on Linux and other
 Unix operating systems it is normally recommended to link against the
 precompiled shared libraries provided by the distributor or vendor.
 
+Some people have offered to provide OpenSSL binary distributions for
+selected operating systems. See the OpenSSL wiki page at [wiki.openssl.org/index.php/Binaries](https://wiki.openssl.org/index.php/Binaries)
```

Comment:
Could we wrap the line at column 80 to avoid this very long line?

Reply:
This should be enough: `<https://wiki.openssl.org/index.php/Binaries>`

https://daringfireball.net/projects/markdown/syntax#autolink

Reply:
For a more modern reference: https://spec.commonmark.org/0.29/#autolinks